src/types/index.ts

```diff
@@ -173,6 +173,30 @@ export type PaymentRequestButtonElementComponent = React.FC<
   PaymentRequestButtonElementProps
 >;
 
+export interface AuBankAccountElementProps extends ElementProps {
+  /**
+   * An object containing [Element configuration options](https://stripe.com/docs/js/elements_object/create_element?type=auBankAccount).
+   */
+  options?: stripeJs.StripeAuBankAccountElementOptions;
+
+  /**
+   * Triggered when data exposed by this Element is changed (e.g., when there is an error).
+   * For more information, refer to the [Stripe.js reference](https://stripe.com/docs/js/element/events/on_change?type=auBankAccount).
+   */
+  onChange?: (event: stripeJs.StripeAuBankAccountElementChangeEvent) => any;
+
+
+  /**
+   * Triggered when the Element is fully rendered and can accept imperative `element.focus()` calls.
+   * Called with a reference to the underlying [Element instance](https://stripe.com/docs/js/element).
+   */
+  onReady?: (element: stripeJs.StripeAuBankAccountElement) => any;
+}
+
+export type AuBankAccountElementComponent = React.FC<
```

Comment:
If merging this after #40, please update to `ReactType`

Reply:
I merged #40 so this should be changed to:

```suggestion
export type AuBankAccountElementComponent = FunctionComponent<
```

Reply:
done